### PR TITLE
Rate limiter to deposit call

### PR DESCRIPTION
**Pull request by kroos47**
- Added rate limit to deposit call
- Added a submodule for avail-js wrapper(for now)

### CODE REVIEW

**Review by prabal-banerjee — APPROVED** [review]

File: .gitmodules
Comment (vthunder) [inline .gitmodules:1]: If you're switching to this api, please fully remove src/avail.js so it doesn't end up as cruft here.

File: src/app.js [hunk 1]
```diff
@@ -22,7 +23,25 @@ client.on(Events.InteractionCreate, async interaction => {
   }
 
   try {
-    // Run the command, passing along the interaction
+    // Run the command, passing along the interaction 
+    if (interaction.commandName == 'deposit') {
+      const userId = interaction.user.id;
+      const now = Date.now();
+      const cooldownAmount = 3 * 60 * 1000;
```
Comment (vthunder) [inline src/app.js:32]: This is 3 minutes, right? Can you add a comment like "// 3 minutes" for clarity?

File: src/commands.js [hunk 2]
```diff
@@ -28,25 +28,28 @@ commands.set('deposit', {
...
+            🌐 ${hyperlink('View in explorer', link)}`
           });
         }
       }});
+      console.log(`transferred 1 AVL to ${dest}`)
     } catch (error) {
       console.error(error);
       interaction.followUp({
-        content: `There was a problem transferring to ${dest}. Kindly report to the Avail Team.`,
+        content: `There was a problem transferring. Kindly report to the Avail Team.`,
```
Comment (vthunder) [inline src/commands.js]: More grammatical: "There was a problem with the transfer. Kindly report to the Avail Team."

**Review by vthunder — CHANGES_REQUESTED** [review]
Approved with noted changes - in particular not leaving around unused code (src/avail.js) is important.

**Pull request merged**